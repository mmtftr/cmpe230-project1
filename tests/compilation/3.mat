print(5)


print(4323874*238)


#dmfmsdklfmsd
#fmdfsf
printsep()
printsep()
print(1234567)