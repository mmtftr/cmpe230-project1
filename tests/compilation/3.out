5
1029082012
------------
------------
1234567